spec/03-plugins/14-request-termination/02-access_spec.lua

```diff
@@ -387,6 +391,10 @@ for _, strategy in helpers.each_strategy() do
           },
           raw_body = 'cool body',
           scheme = 'http',
+          uri_captures = {
+            named = {},
+            unnamed = {}
+          },
```

Comment:
Can we please add a test that has this populated since we are adding a new field to the response?

Reply:
Thought about it, but didn’t. This test already proves that the output of the PDK function gets injected. The tests for the PDK function cover its output. So this is covered. Hence didn’t add it. 

Let me know if extra tests are needed.

Reply:
I expected such a response.
One could replace the code in the plugin handler to output a couple of empty tables under the 'uri_captures' table key and no test would fail. I recommend adding a test or updating an existing test to logically cover as much of the code as possible.

Reply:
added